lambdas/validator.py

```diff
@@ -149,14 +162,22 @@ def ping() -> dict:
         """select count(*) as inventory_count from inventory;"""
     )
 
-    return generate_result_response(
+    return generate_result_http_response(
         {
             "response": "pong",
             "inventory_query_test": count_df.to_dict(orient="records"),
         }
     )
 
 
+def inventory() -> dict:
+    """Validate a single AIP."""
```

Comment:
Think this docstring needs some updating!  😎 

This is a scenario where I think it'd be nice if the docstring outlined what columns to expect in the CSV.  I was going to propose the unit test could actually mock those columns as a way of documenting it... but explicit in the docstring feels the most appropriate.

Reply:
My bad, fixed!